packages/dd-trace/src/appsec/remote_config/manager.js

```diff
@@ -3,11 +3,12 @@
 const { URL, format } = require('url')
 const uuid = require('crypto-randomuuid')
 const { EventEmitter } = require('events')
-const Scheduler = require('./scheduler')
 const tracerVersion = require('../../../../../package.json').version
 const request = require('../../exporters/common/request')
 const log = require('../../log')
+const { getExtraServices } = require('../../service-naming/extra-services')
 const { UNACKNOWLEDGED, ACKNOWLEDGED, ERROR } = require('./apply_states')
+const Scheduler = require('./scheduler')
```

Comment:
why change that now ? haha, totally unrelated to the PR no ?

Reply:
:smiley: related with this https://github.com/DataDog/dd-trace-js/pull/3635#discussion_r1329731420

Reply:
yeah I understand but this is already in the right spot no ? it's the first internal module required after events

Reply:
Now the modules are organized like:
- external
- tracer
- own (appsec)

Reply:
ok whatever